web/src/beta/ui/fields/TripletInputField.tsx

```diff
@@ -0,0 +1,98 @@
+import { FC, useCallback, useState } from "react";
+
+import { TextInput, Typography } from "@reearth/beta/lib/reearth-ui";
+import { styled } from "@reearth/services/theme";
+
+import CommonField, { CommonFieldProps } from "./CommonField";
+
+export type TripletInputFieldProps = CommonFieldProps & {
+  values?: [string, string, string];
+  placeholders?: [string, string, string];
+  disabled?: boolean;
+  onChange?: (values: [string, string, string]) => void;
+  onBlur?: (values: [string, string, string]) => void;
+  content?: [string, string, string];
+};
+
+const TripletInputField: FC<TripletInputFieldProps> = ({
+  commonTitle,
+  description,
+  values = ["", "", ""],
+  placeholders = ["", "", ""],
+  disabled,
+  onChange,
+  onBlur,
+  content = ["", "", ""],
+}) => {
+  const [inputValues, setInputValues] = useState<[string, string, string]>(values);
+
+  const handleChange = useCallback(
+    (index: number, value: string) => {
+      const newValues = [...inputValues] as [string, string, string];
+      newValues[index] = value;
+      setInputValues(newValues);
+      onChange?.(newValues);
+    },
+    [inputValues, onChange],
+  );
+
+  const handleBlur = useCallback(() => {
+    onBlur?.(inputValues);
+  }, [inputValues, onBlur]);
+
+  return (
+    <CommonField commonTitle={commonTitle} description={description}>
+      <InputWrapper>
```

Comment:
Do like the suggestion from [this component](https://github.com/reearth/reearth-visualizer/pull/1017/files#diff-5cbfb66886ee66e502253c94d9af894fe5a690350321580c4b73fd347b4762a5)

Reply:
done